packages/commonwealth/client/scripts/views/components/CommunityStake/useCommunityStake.ts

```diff
@@ -29,6 +29,7 @@ const useCommunityStake = (props: UseCommunityStakeProps = {}) => {
   const ethChainId =
     community?.ChainNode?.ethChainId || app?.chain?.meta?.ChainNode?.ethChainId;
   const activeAccountAddress = app?.user?.activeAccount?.address;
+  const activeChainId = app?.chain?.meta?.ChainNode?.name.split('-')[0];
```

Comment:
can you use ChainNode.id instead of name?
and then compare it against to const 
const BLAST_ID = 1397;

Reply:
This should be resolved